### PR TITLE
Fix model mesh tests and update runtime images

### PR DESCRIPTION
**Pull request by rnetser**
verified via jenkins job/devops/job/rhoai-test-flow/1779/robot/ - failues are not related to this PR and will be fixed in seperate PRs
for the images - tested against disconnected env

### CODE REVIEW

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource [hunk 1]
```diff
@@ -39,8 +40,11 @@ Create Model Server
         ${existing_server}=  Run Keyword And Return Status     Wait Until Page Contains Element    //button[.="${server_name}"]
         IF  ${existing_server}    Run Keyword And Return
         ...    Log    Model Server '${server_name}' already exists, reusing server    console=True
+    ELSE
+        SeleniumLibrary.Click Button    //button[@data-testid="multi-serving-select-button"]
```
Comment (tarukumar) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource:44]: Why you want to hard code  selecting multi-model serving platform?

**Review by tarukumar — DISMISSED** [review]
overall looks good to me. Please run the test from jenkins as well

File: ods_ci/tests/Tests/1000__model_serving/1001__model_serving_modelmesh.robot [hunk 2]
```diff
@@ -202,7 +204,7 @@ Model Serving Suite Setup
     Launch Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
     ...    ${ODH_DASHBOARD_URL}    ${BROWSER.NAME}    ${BROWSER.OPTIONS}
     Fetch CA Certificate If RHODS Is Self-Managed
-    Clean All Models Of Current User
+    Delete Project Via CLI By Display Name    project_title="ALL"
```
Comment (tarukumar) [inline ods_ci/tests/Tests/1000__model_serving/1001__model_serving_modelmesh.robot:207]: This is wrong remove the `" "` from ALL

Comment (rnetser) [reply ods_ci/tests/Tests/1000__model_serving/1001__model_serving_modelmesh.robot]: done

**Review by mwaykole — APPROVED** [review]

**Review by tarukumar — APPROVED** [review]

**Pull request merged**